src/block-components/icon/style.js

```diff
@@ -0,0 +1,298 @@
+/**
+ * External dependencies
+ */
+import {
+	useStyles, getStyles,
+} from '~stackable/util'
+import { compact } from 'lodash'
+import { Style as StyleComponent } from '~stackable/components'
+
+const getStyleParams = ( options = {} ) => {
+	const {
+		selector = '',
+		uniqueId = '',
+		wrapperSelector = '',
+		backgroundShapeSelector = '.stk--shape-icon',
+		hoverSelector = '',
+	} = options
+
+	return [
+		// Icon Styles
+		{
+			selector: `${ selector } svg:last-child`,
```

Comment:
I don't think we need to target `:last-child`
```suggestion
			selector: `${ selector } svg`,
```

Reply:
We need this to avoid resizing the svg with linearGradient tag